### DISCUSSION TITLE
plugin Inquiry

### DISCUSSION THREAD

**Discussion by Stoniasawsome** [Zrips/CMI#3535] [Q&A]
does your plugin cmi support offline server/playernames and or uuids and playernames? im currently using essentials and they dont support offline servers (cracked) and so im starting to have major bugs with the eco and homes. I just wonder do you support both playernames and uuids or just uuids?

**Comment by thboehi** [comment]
I'm using it on a cracked server, and it works as a charm.

**Comment by thboehi** [comment]
As your cracked server will give random UUIDS to players.

**Comment by Stoniasawsome** [comment]
whats your player count?

**Comment by Stoniasawsome** [comment]
i have over 5k when this started happening

**Comment by thboehi** [comment]
I have 1400 players only

**Comment by CloudeLecaw** [comment]
Cmi does Support both. Online and offline players by names and also by UUID. Normaly this works like a charm.
Some commands needed to be the player minimum online on the server one time since your using cmi to work correct.

I cant speak for servers with a big amount like yours but we have tested it for 2350 players there it was fine. So in the theorie it should also work with your 5000+
